docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -425,59 +425,47 @@ Default: `3`
 
 The number of shards per YB-TServer for each YCQL table when a user table is created.
 
-Default: `-1` (the value is calculated at runtime). For servers with up to two CPU cores, the default value is considered as `4`. For three or more CPU cores, the default value is considered as `8`. Local cluster installations, created with `yb-ctl` and `yb-docker-ctl`, use a value of `2` for this flag. Clusters created with `yugabyted` use a default value of `1`.
-
-{{< note title="Important" >}}
-
-This value must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
-
-{{< /note >}}
-
-{{< note title="Important" >}}
-
-When the value is set to *Default* (`-1`), then the server internally *updates* the flag with intended value during startup prior to version `2.18` and the flag remains *unchanged* starting from version `2.18`.
-
-{{< /note >}}
+Default: `-1` (the value is calculated at runtime).
+\
+For servers with up to two CPU cores, the default value is considered as `4`.
+\
+For three or more CPU cores, the default value is considered as `8`.
+\
+Local cluster installations created with `yb-ctl` and `yb-docker-ctl` use a value of `2` for this flag.
+\
+Clusters created with `yugabyted` use a default value of `1`.
```

Comment:
Makes sense to add an empty line before this section as it is not related to explanation of _Default_